src/drivers/sf0x/sf0x.cpp

```diff
@@ -75,6 +76,7 @@
 #include <board_config.h>
 
 #include "sf0x_parser.h"
+#include "tfmini_parser.h"
```

Comment:
Why are you including this on the `sf0x` driver? From what I know this is not a LightWare driver.